linux_os/guide/system/software/updating/security_patches_up_to_date/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_all
+# platform = multi_platform_alinux_multi_platform_anolis,multi_platform_fedora,multi_platform_ol,multi_platform_sle,multi_platform_ubuntu,multi_platform_uos
```

Comment:
there should be a comma instead of underscore after alinux

Reply:
Thank you, fixed.